portworx-manual/Portworx.md

```diff
@@ -40,23 +40,40 @@ You can use the `pxctl` command Portworx provides to obtain metrics on how the v
+        Local Storage Devices: 1 device
+        Device  Path            Media Type              Size            Last-Scan
+        0:1     /dev/sdc2       STORAGE_MEDIUM_MAGNETIC 147 GiB         05 Aug 19 19:02 UTC
+        total                   -                       147 GiB
+        Cache Devices:
+        No cache devices
+        Journal Device:
+        1       /dev/sdc1       STORAGE_MEDIUM_MAGNETIC
+        Metadata Device:
+        1       /dev/sdd        STORAGE_MEDIUM_MAGNETIC
 Cluster Summary
-        Cluster ID: mycluster-00bede14-da12-4df8-88bf-5f73b3f2577e
-        Cluster UUID: c4b28292-2237-4aed-af19-65f9f9b0e125
+        Cluster ID: px-cluster-4148c550-39b2-4954-8a15-fa0cfe584dd8
+        Cluster UUID: 0ee425cf-2131-4a84-b734-152274cd9481
         Scheduler: kubernetes
         Nodes: 3 node(s) with storage (3 online)
-        IP              ID                                      SchedulerNodeName       StorageNode     Used    Capacity        Status  StorageStatus   Version         Kernel                 OS        10.10.1.35      96ad0f14-5bd5-401b-a99e-b8080e135076    aks-default-37257775-0  Yes             9.6 GiB 150 GiB         Online  Up              2.1.2.0-21409c7 4.15.0-1050-azure      Ubuntu 16.04.6 LTS
-        10.10.1.4       7d054b92-c66a-479a-bcd6-2460aac2364c    aks-default-37257775-2  Yes             9.6 GiB 150 GiB         Online  Up              2.1.2.0-21409c7 4.15.0-1050-azure      Ubuntu 16.04.6 LTS
-        10.10.1.66      3b7e8f15-4eed-496b-9cd2-01e30a2393d7    aks-default-37257775-1  Yes             9.6 GiB 150 GiB         Online  Up (This node)  2.1.2.0-21409c7 4.15.0-1050-azure      Ubuntu 16.04.6 LTSGlobal Storage Pool
+        IP              ID                                      SchedulerNodeName               StorageNode     Used    Capacity        Status  StorageStatus   Version         Kernel                  OS
+        10.240.0.6      ce4d4aca-fb23-4ac0-ad36-16fa627f1460    aks-nodepool1-40021484-0        Yes             9.8 GiB 147 GiB         Online  Up (This node)  2.1.1.0-6de97a6 4.15.0-1050-azure       Ubuntu 16.04.6 LTS
+        10.240.0.5      1628bbf7-8780-4d52-9a5b-879f0da3fb14    aks-nodepool1-40021484-1        Yes             9.8 GiB 147 GiB         Online  Up              2.1.1.0-6de97a6 4.15.0-1050-azure       Ubuntu 16.04.6 LTS
+        10.240.0.4      001c6978-7ec9-4c79-b5ca-224c3263738a    aks-nodepool1-40021484-2        Yes             9.8 GiB 147 GiB         Online  Up              2.1.1.0-6de97a6 4.15.0-1050-azure       Ubuntu 16.04.6 LTS
+Global Storage Pool
         Total Used      :  29 GiB
+        Total Capacity  :  441 GiB
 ```
 
 3. **Once your storage class definition is applied and you provision PVCs from your application such as Strimzi**, next grab the Portworx volume list which will give details on the provisioned volumes deployed. In this Kafka example we define the size and replica amount of the volumes in our statefulset yaml for Kafka and zookeeper.
```

Comment:
When we enable volume encryption should we expect to see the "encryption" column from the below command be "YES?

`kubectl exec <portworx_pod_name> -n kube-system -- /opt/pwx/bin/pxctl volume list`

Reply:
Also what is the purpose of Step 4? Just to demonstrate that you can look at the detail of a volume?

Reply:
Yes, Encryption will simply display `Yes` in the command. I can see this being another unit test.
Step 4 allows us to view the volume respective to the node. We can see the bytes used and capacity to each volume. We're also able to view the associated broker consuming the volume.